dbuild/templates/dockerfile.jinja

```diff
@@ -1,4 +1,9 @@
 FROM {{ dist }}:{{ release }}
 MAINTAINER {{ maintainer }}
+{%- if http_proxy is defined %}
+RUN echo 'http_proxy="{{ http_proxy }}"' >> /etc/environment
+RUN echo 'https_proxy="{{ https_proxy }}"' >> /etc/environment
```

Comment:
@sorenh Right. These may or may not be required right now, but it may be in certain build cases where we may want to do wget or curl. 
Example: we encountered one unknown instance of proxy usage while building puppet-rjil where proxy is required for downloading gems.